packages/replay-next/pages/variables.css

```diff
@@ -201,7 +201,7 @@
   --background-color-current-execution-point: #25364e;
   --background-color-current-search-result: #25364e;
   --background-color-warning: #42381f;
-  --background-color-inputs: #0e1119;
+  --background-color-inputs: var(--theme-text-field-bgcolor);
```

Comment:
nit: Since both themes use this same value, may be a good opportunity to collapse two variables?

Reply:
I hear you, but prefer the semantic division here.

Because "inputs" are the superset and "textfields" are a subset, I like the idea of the code referring to inputs as inputs and textfields as textfields. It's true that we currently are using the same color between the two, but it's not uncommon to want to apply a border to all textfields but not also get things like checkboxes or forms.

If we do end up changing this out, I'd propose we add it to a separate cleanup ticket.